index.js

```diff
@@ -32,8 +34,9 @@ module.exports.s3 = function(config, key, args, callback) {
 		);
 	}
 	var s3 = regions[config.region];
+	var s3_request = authenticatedRequest ? s3.makeRequest : s3.makeUnauthenticatedRequest
```

Comment:
can this ben named `s3Request` or similar to stick with the camelCasing

Reply:
Sure! Sorry, I'm currently working a lot with python and used snake_case because of that! hahaha